docs/src/man/getting_started.md

```diff
@@ -1053,15 +1132,18 @@ The CSV.jl functions are not loaded automatically and must be imported into the
 using CSV
 ```
 
-A dataset can now be read from a CSV file at path `input` using
-```julia
-DataFrame(CSV.File(input))
-```
+You can read a .csv file containing your dataset at file pathname _fpath_ using (Windows example):
 
-A `DataFrame` can be written to a CSV file at path `output` using
-```julia
-df = DataFrame(x = 1, y = 2)
-CSV.write(output, df)
 ```
+fpath = "C:\\Users\\biff\\Documents\\myData\\mydataset.csv"
+df = CSV.read(fpath,copycols=true;dateformat="mm/dd/yyyy") 
+```
+**Note** that if you don't include _copycols=true_, then your DataFrame will be **built with immutable, read-only, CSV.Column vectors.** If you only want a read-only table, set _copycols=false_. However, you likely want to mutate data in your DataFrame, so be sure to include _copycols=true_. See the [CSV.jl documentation](https://juliadata.github.io/CSV.jl/stable/) for other input options, such as _dateformat_, which will help get your data read in correctly. 
```

Comment:
This note should be about choosing either `DataFrame` or `DataFrame!` as a constructor from `CSV.File`.